apps/vaporgui/PColorSelector.cpp

```diff
@@ -0,0 +1,36 @@
+#include "PColorSelector.h"
+#include "QColorWidget.h"
+#include <vector>
+#include <vapor/ParamsBase.h>
+
+PColorSelector::PColorSelector(const std::string &tag, const std::string &label)
+: PLineItem(tag, _colorWidget = new QColorWidget)
+{
+    connect(_colorWidget, SIGNAL(colorChanged(QColor)), this, SLOT(colorChanged(QColor)));
+}
+
+QColor PColorSelector::VectorToQColor(const std::vector<double> &v)
+{
+    if (v.size() != 3) return QColor::fromRgb(0,0,0);
+    return QColor::fromRgb(v[0]*255, v[1]*255, v[2]*255);
+}
```

Comment:
It seems the passed in vector `v` might be invalid, given the size test.
Should we also test its values? I.e., are they all between 0.0 and 1.0?